iphone/Classes/DeMarcelpociotCollectionviewCollectionView.h

```diff
@@ -8,7 +8,6 @@
 #import "TiUIView.h"
 #import "DeMarcelpociotCollectionviewCollectionViewProxy.h"
 #import "CHTCollectionViewWaterfallLayout.h"
-#import "M13ContextMenu.h"
```

Comment:
"contextMenu"  is here since the first days, is a feature of this module ppl use this, I do not want just remove this and have ppl angry on me.